### DISCUSSION TITLE
This plugin is not being discovered by tailwind in nextjs project

### DISCUSSION THREAD

**Discussion by chidimo** [tailwindlabs/tailwindcss#7139] [Help]
## Note: Please delete or close this discussion. The plugin actually works but somehow the class is not accessible in the browser.

So I added this plugin to my nextjs app but the classes are not being generated when I run the project with `next dev`. Even the classes for the tailwindcss/forms is not being generated as well. I don't know what I'm doing wrong

My `tailwind.config.js` file content is

```
const colors = require('tailwindcss/colors');

module.exports = {
  content: ['./pages/**/*.{js,ts,jsx,tsx}', './components/**/*.{js,ts,jsx,tsx}'],
  darkMode: false, // or 'media' or 'class'
  theme: {
    extend: {
      colors: {
        'autumn-white': {
          DEFAULT: '#FCFCFA',
        },
        'autumn-yellow': {
          10: '#F7F4EB',
          DEFAULT: '#FAF8F3',
        },
        green: colors.emerald, // need this now that we're on tailwind v3
      },
    },
  },
  variants: {
    extend: {},
  },
  plugins: [require('@tailwindcss/forms'), require('@tailwindcss/line-clamp')],
};
```

I have a `globals.css` file with the below content

```
@tailwind base;
@tailwind components;
@tailwind utilities;

#__next {
  height: 100%
}
```

I'm not so sure if this is an issue with the plugin or with tailwind itself.

Thanks for any help.

**Comment by me-nkr** [comment]
can you tell me more about the issue? I am facing a kind of similar issue here : /
